OpenRA.Mods.Common/Scripting/Global/MapGlobal.cs

```diff
@@ -108,7 +108,7 @@ public string TerrainType(CPos cell)
 		}
 
 		[Desc("Returns true if there is only one human player.")]
-		public bool IsSinglePlayer { get { return Context.World.LobbyInfo.IsSinglePlayer; } }
+		public bool IsSinglePlayer { get { return Context.World.LobbyInfo.IsSingleHuman; } }
```

Comment:
Similarly here: Having lua scripts behave differently for the same single *player* game depending on whether or not there are spectators is a bug.  `LobbyInfo.NonBotPlayers.Count() == 1`